src/gauravk/Assignment_4/AsciiOfCharacter.java

```diff
@@ -0,0 +1,29 @@
+package gauravk.Assignment_4;
+/*#4
+
+Program 1:  Write a method which will return ascii value of given character.
+Hint : Method signature should be int getAsciiOfChar(char ch)
+*/
+
+import java.util.Scanner;
+
+public class AsciiOfCharacter {
+	int asci;
+	void userInput() {
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Type a character to find its ASCII value.");
+		char c = sc.next().charAt(0);
+		AsciiOfCharacter asciiOfCharacter2 = new AsciiOfCharacter();
```

Comment:
we can directly call non-static method from non-static method, not need to create object of class